crates/client/eth/Cargo.toml

```diff
@@ -0,0 +1,70 @@
+[package]
+authors.workspace = true
+description = "This crate is responsible to handle l1 communication"
+edition = "2021"
+homepage = "https://github.com/KasarLabs/deoxys"
+license = "MIT"
+name = "dc-eth"
```

Comment:
rename it to something more meaningful `dc-eth-core-contract`

Reply:
I prefere `dc-eth` actually since it concerns the eth client itselfs and may bring together other eth acceses than the the core contract